### ISSUE TITLE
Boundary sounds (play area and shape limits)

### ISSUE THREAD

**Issue by BLFiedler** [phetsims/quadrilateral#127]
We'd like to come up with two different sounds that indicate a limit has been reached when manipulating the shape. 

1.) the Play Area boundary is limiting shape transformation (a vertex has collided with the boundary, even if the vertex is under direct control)

2.) A vertex has collided with a side or vice versa. The side limit for minimum length has been reached.

We have examples of these in other sims:
Faraday's Law (1)
Gravity Force Lab Basics (1 & 2)
Balloons and Static Electricity (1)

**Comment by jessegreenberg** [comment]
Yep, we can do that. I never added observables that notify when we hit against shape or model boundaries, but I can do that as part of this issue.

**Comment by jessegreenberg** [comment]
OK, this is done. @BLFiedler over to you to have a listen.

**Comment by BLFiedler** [comment]
Edge boundary sounds are good! Some work to happen in conjunction with #270 to allow them to repeat at new edges.

We would like the sound that plays when the vertex or side movement is blocked by the shape itself.

@Ashton-Morris, a new task for you! The subtleties here are that the shape never actually "collides" with itself. It often stops short. So, a sound that is overly physical may not make sense here. Not to say it can't be based on a physical sound, but should probably be abstracted a little. also, it's possible you'll be in a situation (e.g., concave shape) where you could hear this a lot. So it should be shouting "NO", so much as whispering it. If that makes sense.

@terracoda suggested abstracting the following
- Door shut
- Cracking

@BLFiedler suggested some less harsh variant of the usual UI sounds you hear when you have done something you can't do.

**Comment by Ashton-Morris** [comment]
I think I understand this properly thanks @BLFiedler I have added some sounds to [this folder to try.](https://drive.google.com/drive/folders/1sTyhfR9EZVJWl7GTfi3COXGHK9yCqoa8?usp=share_link)

- quad-edge-boundary-v2.mp3
- quad-edge-boundary-v3.mp3
- quad-edge-boundary-v4.mp3
- quad-edge-boundary-v5.mp3

**Comment by BLFiedler** [comment]
Thanks, @Ashton-Morris!

Let's go with the least physical sound for right now. I think v5 could work here. It's subtler than the typical "DONT GO THERE" sound, which is great. The next favorite was v2, which we can try after if v5 doesn't pan out.

Also, Consultant working with @terracoda noted that the edge boundary sound could be a bit louder. Let's raise it by say 50%.

@jessegreenberg ,
- [ ] Differentiate outer edge and inner shape boundary. Assign inner shape boundary to `quad-edge-boundary-v5.mp3` as a first go.
- [ ] Raise outer edge boundary sound by 50% in volume

**Comment by jessegreenberg** [comment]
OK! This sound was added for these cases. @BLFiedler @Ashton-Morris would you like to review?
